apps/frontend/src/utils/aave/AaveUserReservesSummary.ts

```diff
@@ -209,9 +209,10 @@ export class AaveUserReservesSummaryFactory {
           );
 
           // how much the user can borrow if there's no limit of supply
-          const canBorrow = borrowPower
+          let canBorrow = borrowPower
             .sub(borrowBalance)
             .div(r.reserve.priceInUSD);
+          canBorrow = canBorrow.gt(0) ? canBorrow : Decimal.ZERO;
```

Comment:
```suggestion
          const canBorrow = borrowPower
            .sub(borrowBalance)
            .div(r.reserve.priceInUSD)
            .max(Decimal.ZERO);
```
you can use `max` to directly handle the condition of whether the result is greater than zero